src/main/resources/db/changelog/20201103112757-purge-current-sca-certs.xml

```diff
@@ -6,7 +6,7 @@
         xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
         http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
 
-    <changeSet id="20201103112757-1" author="crog">
+    <changeSet id="20201103112757-1" author="sdhome">
```

Comment:
Convention on this is to have both authors listed, delimited with a comma: `"crog, sdhome"`

Reply:
ack.